WooCommerce/Classes/Extensions/UINavigationController+Woo.swift

```diff
@@ -57,30 +57,20 @@ extension UIViewController: UINavigationBarBackButtonHandler {
 }
 
 extension UINavigationController: UINavigationBarDelegate {
-
-    // This delegate method is not called on the simulator or device running iOS 13.4 from Xcode.
-    // You need to use a release build.
-    public func navigationBar(_ navigationBar: UINavigationBar, shouldPop item: UINavigationItem) -> Bool {
-        guard let items = navigationBar.items else {
-            return false
-        }
-
-        if viewControllers.count < items.count {
-            return true
-        }
-
+    public func checkIfNavigationBarShouldPop(_ navigationBar: UINavigationBar, item: UINavigationItem) -> Bool {
```

Comment:
nit: Could we simplify this method since we aren't using its parameters (assuming we don't add any additional checks here)?

```suggestion
    public func checkIfNavigationBarShouldPop() -> Bool {
```